guides/deployment/deployment.md

```diff
@@ -69,7 +69,7 @@ To run Phoenix in production, we need to set the `PORT` and `MIX_ENV` environmen
 
 ```console
 $ PORT=4001 MIX_ENV=prod mix phx.server
-10:59:19.136 [info] Running MyAppWeb.Endpoint with Cowboy on http://example.com
+10:59:19.136 [info] Running MyAppWeb.Endpoint with Cowboy on https://example.com
```

Comment:
I do not think we should change this URL– the current value matches the default Endpoint configuration from the Phoenix installer.
```suggestion
10:59:19.136 [info] Running MyAppWeb.Endpoint with Cowboy on http://example.com
```